issue2pr.py

```diff
@@ -9,10 +9,27 @@
 
 from __future__ import print_function
 
+import argparse
+import getpass
+import json
```

Comment:
Can you?  I haven't used requests much, but the only example I saw in the docs had it explicitly converting POST data to JSON.